src/sentry/lang/native/minidump.py

```diff
@@ -50,6 +55,15 @@ def process_minidump(minidump, cfi=None):
         return ProcessState.from_minidump_buffer(minidump, cfi)
 
 
+def _guess_frame_quality(frames):
```

Comment:
This is the whacky part I am not sure about. It scores a stacktrace's quality.

Reply:
Let's remove this, and as @jan-auer proposed, only let users overwrite the crashing thread's stacktrace.